src/main/java/pl/gov/coi/pomocua/ads/dev/FakeUserRepository.java

```diff
@@ -0,0 +1,13 @@
+package pl.gov.coi.pomocua.ads.dev;
+
+import pl.gov.coi.pomocua.ads.UserId;
+import pl.gov.coi.pomocua.ads.users.User;
+import pl.gov.coi.pomocua.ads.users.UsersRepository;
+
+public class FakeUserRepository implements UsersRepository {
+
+    @Override
+    public User getById(UserId userId) {
+        return new User(new FakeCurrentUser().getCurrentUserId(), "fake@email.invalid");
```

Comment:
`FakeUserRepository` is a spring bean, so maybe we could inject `CurrentUser` here?
I know that both of them are plugged in with the same `dev` profile, but `FakeUserRepository` doesn't have to know that